### ISSUE TITLE
Recognizing Rankings by Gender

### ISSUE THREAD

**Issue by lgarron** [thewca/wca-regulations#160]
tl;dr: Have a filter to view female rankings, but no labeled female (world/regional) records.

---

[This thread](http://www.speedsolving.com/forum/showthread.php?44568-Proposal-Separate-WCA-Rankings-Officially-by-Gender) is seeing a lot of new discussion.
This is not technically under the scope of the WRC, but it lies close to to 9i1. Also, the WRC is currently the most appropriate for things that require community discussion.
I used to be concerned about making such rankings public, but I think there is significant interest in the community (there are various ways it's being tracked unofficially), and there aren't objections from females based on technical/personal/moral/privacy grounds.
- Gender has been public WCA data for a long while, and there haven't been strong objections to that.
- Many people (women/girls and general community) are interested.
- This is _not_ an arbitrary category.
  - To [cite Stefan](http://www.speedsolving.com/forum/showthread.php?44568-Proposal-Separate-WCA-Rankings-Officially-by-Gender&p=914642&viewfull=1#post914642)
    - Females are very much underrepresented (half the general population but only 10% in cubing).
    - Major cultural precedent for gender distinction.
    - Some people want it.
  - Additionally, I believe that this is worthwhile if it encourages female cubers (either current ones, or potential ones because there will be role models/examples to show that this sport is for them, too).
### Implementation

I believe the following is reasonable on the WCA results:
- A filter to view female-only results in just about any place you can already view by region.
- _No_ officially recognized "Female Record" label (nor "Male Record", nor any version of these by region).
  - There will always be the fastest official female, just like there will be the fastest person with the initials LG, or the fastest non-lucky 3x3x3 solve, or the youngest BLD solver. Informally, a record is any result that is superlative according to some measure. Just because we make it easier to see certain rankings/records does not mean we have to award a label to it, or prevent people from unofficially referring to things like "female record".
- We will need to be sensitive about gender preferences.
  - Rather than requiring birth certificates or imposing some normative judgment, I think the best is to allow self-selected gender.
    - We can also ask for a reason if the specified gender doesn't match ID/passport. However, many competitions don't ask for ID if you're competing with the local nationality, and I don't think we should require more use of ID (especially for a particular gender).
    - If someone claims to be female just to be at the top of the female rankings, the Board can probably have a word with them.
  - We should have an "other" or "explicitly unspecified" category. I don't think we should be too accommodating, but there should be an opt-out of binary gender.
  - We will need a way to deal with unknown gender data. It's probably simplest/safe to exclude them from any gender rankings. (e.g. the 4th-fastest female will be the 4th fastest of those whom we have recorded as female)
### Some Thoughts
- I think we should make an effort to get input from as many female cubers as possible, _internationally_. It is not a problem if they are biased, but since this change is clearly about them we should listen to their wishes/concerns more than to those of males.
- I don't think we should listen to arguments of the form "girls just want this so they can get records more easily". We are aware that this is part of the point, and gender rankings aren't an arbitrary grab for recognition.
  - I also don't think we should have male rankings just "to be fair/equal". (We might still want male rankings for other reasons, e.g. to make the interface consistent/exhaustive.)
- Making gender-based information more visible may be either against either personal preference or privacy concerns.
  - I used to have some concerns, and Dene still has some. The discussion isn't really centered around privacy right now.
  - The strongest concern I can currently think of is that early competitors (before, say, 2010) might not desire to be highlighted as female on web results for their name when they didn't explicitly give us permission to publish their gender. But we already _do_ publish gender on the personal pages, and new competitors give their consent in [2d](https://www.worldcubeassociation.org/regulations/#2d).
  - Note: Unlike birthdate, gender not especially sensitive [PII](https://en.wikipedia.org/wiki/Personally_identifiable_information) given that the _names are already public_ (i.e. the name already indicates the gender most of the time, so gender isn't really private).
  - A thought: Just because we haven't heard concerns about this doesn't mean no one cares. Especially if they haven't noticed, e.g. because they assumed it wouldn't happen.
- This will of course take some work on the results system. Preferably the new one.
- We need a good _official_ answer for [why we don't do age divisions](http://www.speedsolving.com/forum/showthread.php?35119-What-is-a-good-explanation-for-why-the-WCA-does-not-have-age-categories) (or... start having them?).
- We should start being clear about the intended use of gender information, during comp. registration.
- In many places, this would be the first way to view rankings of a subset of competitors without a labeled record (WR/CR/NR) at the top.
  - This is going down the path of mixing official results with unofficial views/rankings/statistics. The [statistics page](https://www.worldcubeassociation.org/results/statistics.php) has let us keep them separate for a long while.

**Comment by Hays10** [comment]
Kevin Hays here.

I'm pretty indifferent on the addition of adding these rankings, as long as they aren't implemented on the main rankings page, but instead put in the Statistics page. Lucas mentioned that the statistics page has allowed fun and official rankings to remain separate and I think it's very important it stays like this.

Being on the top of the rankings page should be reserved for those with a WR/CR/NR and allowing gender rankings on the main rankings page makes the official rankings less special. It takes countless hours of hard work to make it to the top of the rankings page, and I would feel cheated if someone else was able to do the same with much less effort.

**Comment by ghost** [comment]
Hey, heard/seen nothing about any of this! Just wanted to comment for the sake of wondering if this will happen at all or if any discussion will be done about it?

Also: "We should have an "other" or "explicitly unspecified" category. I don't think we should be too accommodating, but there should be an opt-out of binary gender." I truly think this is an important issue the WCA should take a progressive stance on. These sort of gender specifications are on Facebook and I'm sure will gain more acceptance as time goes on. Since the WCA is not a gender segregated sport, the change provides no complications for the WCA. There are competitors who would be more likely to compete if specifying a gender was not required :)

**Comment by lgarron** [comment]
> Just wanted to comment for the sake of wondering if this will happen at all or if any discussion will be done about it?

Unfortunately, this didn't happen because no one on the website team had time to implement it. That might change soon, but not very soon. Talk to @jfly if you are interested in taking a crack at implementing it in https://github.com/cubing/worldcubeassociation.org

> There are competitors who would be more likely to compete if specifying a gender was not required :)

I don't deny this, but do you have particular examples in mind? Motivating examples are useful. :-)

**Comment by ghost** [comment]
Just today Joshua Block (supercavitation on SS and #) was saying his friend would compete but they do not consider themselves male or female so that has discouraged them. Cameron Palone (hawkmp4 on SS) recently regained interest in cubing but decided not to because the WCA would not alter gender (also names but this is a separate issue). Those are just two personal stories I have, I'm sure others have their own.

And all that's stopping the female ranks is some coding? That can be arranged :)

**Comment by Laura-O** [comment]
As we also discussed recently, this is not a topic the WRC can decide, so I am closing this here.

**Comment by jfly** [comment]
> "We should have an "other" or "explicitly unspecified" category.

For the record, we do have that now. See https://github.com/thewca/worldcubeassociation.org/pull/1542.

> As we also discussed recently

Where was that discussion?

> this is not a topic the WRC can decide

Who *does* have the power to make this decision? Where should this discussion continue?

**Comment by Laura-O** [comment]
Oops, sorry, I had this issue open at the same time as the issue about releasing age information and confused them. Thanks for noticing that @jfly.

On a different note, I don't see what the WRC can do here. Please let me know if I am missing something, but I can't see why or how this could be added to the regulations.

**Comment by jfly** [comment]
> I can't see why or how this could be added to the regulations.

I could see this as an addition underneath [9i](https://www.worldcubeassociation.org/regulations/#9i). If, at some point, if we decide to show rankings filtered by gender, but do not award corresponding "world records", then I'd like for that to be made explicit somewhere, and 9i might be a good place for it.

More important than "should this be a regulation or not" is just the fact that this discussion really should have the opportunity to happen somewhere. AFAIK, only the WRC and the WST are technologically capable of having public discussions, so I'd argue it should happen on either this github issue, or over on https://github.com/thewca/worldcubeassociation.org/issues/168. I don't particularly care which one =)

I think another good question to ask is "how exactly *could* this change happen? The WST could certainly just make the change to the website, but presumably the rest of the WCA should be consulted on this. I could see 4 ways to do it:

1. The WST just makes the change.
2. The WRC updates the regulations to say that the WCA will list rankings by gender, the Board approves the change to the regulations, and the WST implements it.
3. The Board just tells the WST to make the change.
4. Delegates vote on the issue and if a majority is for it, the WST makes the change.

**Comment by Laura-O** [comment]
I think we should handle this as a "website feature" and not add it to the regulations. We also have annual rankings but we do not recognize annual world records.

> 4. Delegates vote on the issue and if a majority is for it, the WST makes the change.

That's definitely the way I would prefer to do this.
I have started to prepare the next WRC polls yesterday, so it's a good opportunity now. I will add this to the polls, so we can finally resolve this issue.

**Comment by ghost** [comment]
Happy to see this getting some attention 👍

When I was competing last month I thought of this again, I look forward to seeing the outcome of this vote.

Thanks again!

**Comment by Laura-O** [comment]
Results of the poll:

_Should the WCA website allow filtering rankings by gender?_
Yes - 84 votes -73.04%
No - 31 votes - 26.96%

**Comment by lgarron** [comment]
This will be a subtle change to convey. I posted about this privately on Facebook, and the discussion reiterated to me that we need to remain careful to announce this as gender filtering for rankings, not "female rankings" (and certainly not "female records").

@growan, you posted about this a bunch in the past: https://www.speedsolving.com/forum/threads/proposal-add-gender-sorting-on-the-wca-rankings.44568/#post-914543

In particular, you made a comparison to chess:
> I personally do not think Official World Records should be split, but I do think having quick access on the WCA website to the top 100 female cubers would be extremely beneficial to female cubers. A debate like this has also been had amongst the Chess community, not surprisingly, and I apologize that a lot of my references are about Chess, there isn't much data on Rubik's Cubing in sociological or psychological fields, but the two sports do have quite a bit in common in terms of how they are typed as intellectual and their need for spatial intelligence and reasoning skills. I can't say why the speed stacking is split amongst genders, but Chess is split amongst genders to encourage female participation.

I've brainstormed at https://github.com/thewca/worldcubeassociation.org/issues/1919 about how we might convey that there are no official female rankings.

If you have more thoughts, or citations for similar decisions and rankings in other sports, I think it would be helpful to have those in mind as the WCA implements and announces the feature.

@Laura-O: Do you mind keeping this bug open, or would you prefer to move future discussion to https://github.com/thewca/worldcubeassociation.org/ ?

**Comment by Laura-O** [comment]
> @Laura-O:  Do you mind keeping this bug open, or would you prefer to move future discussion to https://github.com/thewca/worldcubeassociation.org/ ?

As we are discussing how to implement this now, it might make sense to move this to https://github.com/thewca/worldcubeassociation.org/.

**Comment by UnderwaterCuber** [comment]
Would this also allow be opening up things like men’s and women’s records or would it just be allowing people to sort rankings by gender?
